build/build.sh

```diff
@@ -91,12 +91,11 @@ build(){
                 output_name+='.exe'
             fi
             echo "Building for: ${GOOS} ${GOARCH}"
-            gox -cgo -os="$GOOS" -arch="$GOARCH" -ldflags \
-                "-X github.com/openebs/node-disk-manager/pkg/version.GitCommit=${GIT_COMMIT} \
-                -X main.CtlName='${CTLNAME}' \
-                -X github.com/openebs/node-disk-manager/pkg/version.Version=${VERSION}" \
-                -output="$output_name" \
-                ./cmd/"$BUILDPATH"
+            go build \
```

Comment:
Don't we need to set the `CGO_ENABLED` env, or is it by default set to `1`. Can we check the default value of `CGO_ENABLED`

Reply:
CGO_ENABLED=1 is set in script. Thanks for pointing it out @harshthakur9030